src/kernel.jl

```diff
@@ -328,6 +329,67 @@ function Base.convert(::Type{AbstractArray}, L::Laplacian{N}) where N
 end
 _reshape(L::Laplacian{N}, ::Type{Val{N}}) where {N} = L
 
+
+"""
+    gabor(size_x,size_y,σ,θ,λ,γ,ψ) -> (k_real,k_complex)
+
+Returns a 2 Dimensional Complex Gabor kernel contained in a tuple where
+
+    size_x,size_y is the size of the kernel;
```

Comment:
can you make those into bullet points, and surround the parameters in backticks?

e.g.
```
- `size_x`, `size_y` denote the size of the kernel
```
etc